sdk/search/search-documents/samples/v12-beta/typescript/src/searchClientOperations.ts

```diff
@@ -85,7 +85,7 @@ async function main(): Promise<void> {
     // from the client constructor. In that case, you can use the `string[]` type.
     const select = ["hotelName"] as const;
 
-    const searchResults = await searchClient.search("luxury", {
+    const searchResults = await searchClient.search("luxury", undefined, undefined, {
```

Comment:
These files should be generated using `npx dev-tool samples publish`. After fixing the method signature and reverting the changes in `samples-dev`, run that script to generate the samples in this `samples` directory.